src/eXpansion/Framework/GameCurrencyBundle/Plugins/Gui/BillWindow.php

```diff
@@ -137,8 +150,8 @@ function () use ($manialink, $bill) {
                     [], "Success", 3500, $manialink->getUserGroup());
                 $this->closeManialink($manialink);
             },
-            function ($status) use ($manialink) {
-                $this->notifications->error("Server said: ".$status,
+            function ($error) use ($manialink) {
```

Comment:
when you arguments of a function do'nt fit one line you put them all in a new line now half of them.

```php
<?php
$this->notifications
	->error(
		"Server said: ".$error,
		[], 
		"Error", 
		10500, 
		$manialink->getUserGroup()
	);
```